GeoHealthCheck/app.py

```diff
@@ -468,95 +469,112 @@ def add():
+                return redirect(url_for('add', lang=g.current_lang))
+
+        tags_to_add = []
+        for tag in chain(tags, resource_tags):
+            tag_obj = tag
+            if not isinstance(tag, Tag):
+                tag_obj = Tag.query.filter_by(name=tag).first()
+                if tag_obj is None:
+                    tag_obj = Tag(name=tag)
+            tags_to_add.append(tag_obj)
+
+        resource_to_add = Resource(current_user,
+                                   resource_type,
+                                   title,
+                                   resource_url,
+                                   tags=tags_to_add)
+
+        resources_to_add.append(resource_to_add)
+        probe_to_add = None
+        checks_to_add = []
+
+        # Always add a default Probe and Check(s)
+        # from the GHC_PROBE_DEFAULTS conf
+        if resource_type in CONFIG['GHC_PROBE_DEFAULTS']:
+            resource_settings = CONFIG['GHC_PROBE_DEFAULTS'][resource_type]
+            probe_class = resource_settings['probe_class']
+            if probe_class:
+                # Add the default Probe
+                probe_obj = Factory.create_obj(probe_class)
+                probe_to_add = ProbeVars(
+                    resource_to_add, probe_class,
+                    probe_obj.get_default_parameter_values())
+
+                # Add optional default (parameterized)
+                # Checks to add to this Probe
+                checks_info = probe_obj.get_checks_info()
+                checks_param_info = probe_obj.get_plugin_vars()['CHECKS_AVAIL']
+                for check_class in checks_info:
+                    check_param_info = checks_param_info[check_class]
+                    if 'default' in checks_info[check_class]:
+                        if checks_info[check_class]['default']:
+                            # Filter out params for Check with fixed values
+                            param_defs = check_param_info['PARAM_DEFS']
+                            param_vals = {}
+                            for param in param_defs:
+                                if param_defs[param]['value']:
+                                    param_vals[param] =\
+                                        param_defs[param]['value']
+                            check_vars = CheckVars(
+                                probe_to_add, check_class, param_vals)
+                            checks_to_add.append(check_vars)
+
+        result = run_test_resource(resource_to_add)
+
+        run_to_add = Run(resource_to_add, result)
+
+        DB.session.add(resource_to_add)
+        if probe_to_add:
+            DB.session.add(probe_to_add)
+        for check_to_add in checks_to_add:
+            DB.session.add(check_to_add)
+            DB.session.add(run_to_add)
 
     try:
         DB.session.commit()
-        msg = gettext('Service registered')
+        msg = gettext('Services registered')
```

Comment:
Do we need to update our `.po` files?

Reply:
Right. Will update.

Reply:
That is strange, i've extracted messages, and pot is updated correctly. however pybabel update won't update .po files with new strings. Any idea what's wrong?

Reply:
Ok, I found the cause, `update_translations` in paver script was using wrong path for template. Fixed in this branch